docker-compose.yml

```diff
@@ -8,3 +8,4 @@ services:
         - '5900:5900'
       volumes:
         - '/lib/modules:/lib/modules:ro'
+        - './passwd:/home/user/.vnc'
```

Comment:
This will result in creating an empty `.vnc` when `./passwd` is missing on the host, so this seems to cause a security issue.

Reply:
pretty sure it will be populated afterwards, at least as far as i have been able to test